File: codeforces/group_bsu/problems/t/t.py
# n, bal = [int(i) for i in input().split()]
# print(bal - ((n//2)*(2 + n-1)))

n, bal = [int(i) for i in input().split()]
i = 1
while True:
    if bal >= i:
        bal-=i
    else:
        print(bal)
        break
    i+=1
    if i > n:
        i=1

File: codeforces/group_bsu/problems/t/t.c
#include <stdio.h>

int main() {
    int n, i = 1;
    long long int bal;
    // scanf("%d %d",&n, &bal);
    n = 11;
    bal = 2147483648;
    // bal = bal % (int)(((float)n/2)*(2 + n - 1));
    while (1){
        if (bal>=i){
            bal-=i;
        } else{
            printf("%d\n", bal);
            break;
        }
        i+=1;
        if (i>n) {
            i=1;
        }

    }

    return 0;
}